Cargo.toml

```diff
@@ -23,7 +23,7 @@ path = "src/driver.rs"
 [dependencies]
 clippy_lints = { path = "clippy_lints" }
 semver = "1.0"
-rustc_tools_util = "0.2.1"
+rustc_tools_util = {version = "0.2.1", path = "./rustc_tools_util"}
 tempfile = { version = "3.2", optional = true }
 termize = "0.1"
```

Comment:
I think we need to revert this for now, otherwise in the rustc repo we might get some tidy error about duplicate dependencies between clippy and miri?

Reply:
The problem is that Clippy won't compile without it as the made changes are incompatible with v0.2.1 or at least produce a warning, which we forbid in the CI. I think the best way would be to release `v0.3.0` from this branch and then change the dependency to that as well. Then we can merge it without any problems.

I'll wait on your feedback, before addressing the import thing :)